cmd/crio/main.go

```diff
@@ -46,6 +48,29 @@ func validateConfig(config *server.Config) error {
 	return nil
 }
 
+func parseIDMap(spec []string) (m [][3]int, err error) {
```

Comment:
I think this should be a common function in either storage or in libpod/pkg  so that buildah, podman and cri-o can easily share.